docs/getting-started/features.md

```diff
@@ -1,9 +1,7 @@
 ---
-sidebar_position: 3
+title: Features
 ---
```

Comment:
I like the idea of having a features page like this.

Two questions come to mind:

1. Should this be incorporated to the intro?
2. Why is features listed under quickstart and not at the top level? Quickstart is when someone has decided to try it. Aren't features looked at when someone is still trying to make up their mind?

Reply:
I thought the `Features` section is too elaborate to fit into the introduction page as we just provide a high level purpose of the app in the `Introduction`.. Also, do you mean `Getting Started` by `quickstart`, because we do not have a quickstart.. Just to make sure we both are looking at the latest commit, here's the sidebar menu structure I see..

![image](https://user-images.githubusercontent.com/59538726/154160530-f39eb2ff-1311-4418-9f2c-7256bc64bf8f.png)